src/components/SQForm/SQFormDatePicker.tsx

```diff
@@ -41,7 +72,9 @@ function SQFormDatePicker({
     onChange
```

Comment:
useForm calls onChange differently than we are using it in handleChange. Is there some typing work here that needs done? This was one detail that held me up a bit when I was duplicating this work.

Reply:
@hvilander `useForm `use `React.ChangeEventHandler` type on `onChange` which is generic type. `useForm `didn't specifies the arguments type for `onChange` .  I am defining types of those arguments specifically and other than that the type of function remains the same. `useForm` is being used in almost every component So I think those component have their different type of arguments in `change. That is why they didn't specify the argument type in `useForm` itself